### DISCUSSION TITLE
Maps not showing

### DISCUSSION THREAD

**Discussion by Studio88-KM** [code4recovery/12-step-meeting-list#1033] [General]
Hey Josh, any idea why the maps aren't showing on the individual meetings? They haven't been showing since the site went live on 12/22 ish or so. 
I've checked other browsers and cleared the website and server cache. 
The correct URL is in mapbox and it's the correct access token. 

https://danyc.org/meetings/

**Comment by joshreisner** [comment]
🤔 looks ok to me!

<img width="1393" alt="Screenshot 2023-01-03 at 1 08 46 PM" src="https://user-images.githubusercontent.com/1551689/210441879-00e0bc19-adcc-49ea-b483-19afaf596405.png">

**Reply by Studio88-KM** [reply]
It's back up for us too. Thanks Josh!
